util/fibers/epoll_socket.cc

```diff
@@ -352,7 +352,7 @@ auto EpollSocket::WriteSome(const iovec* ptr, uint32_t len) -> Result<size_t> {
 
     DCHECK_EQ(res, -1);
 
-    if (res != EAGAIN) {
+    if (errno != EAGAIN) {
```

Comment:
You even `DCHECK_EQ(res, -1);` above :rofl:

Reply:
ouch.

Reply:
You have it right in other places I checked :smile: